packages/loot-design/src/components/modals/ImportTransactions.js

```diff
@@ -380,6 +394,22 @@ function DateFormatSelect({
   );
 }
 
+function MultipliersOption({ value, onChange }) {
+  return (
+    <View
+      style={{
+        flex: 1,
+        flexDirection: 'row',
+        alignItems: 'center',
+        userSelect: 'none',
+      }}
+    >
+      <Checkbox id="add_multiplier" checked={value} onChange={onChange} />
+      <label htmlFor="add_multiplier">Add Multiplier</label>
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏love the attention to detail with `id` and `htmlFor`!